referenceframe/frame.go

```diff
@@ -501,7 +501,9 @@ func (rf *rotationalFrame) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
-	rf.baseFrame = &baseFrame{name: rf.Name(), limits: []Limit{{Min: cfg.Min, Max: cfg.Max}}}
+	rf.baseFrame = &baseFrame{name: cfg.ID, limits: []Limit{
+		{Min: utils.DegToRad(cfg.Min), Max: utils.DegToRad(cfg.Max)},
+	}}
```

Comment:
what is the bug here?

Reply:
nope.  This is fixing a bug that was previously introduced